commands/song.js

```diff
@@ -0,0 +1,15 @@
+const {MessageEmbed, Message} = require("discord.js");
+
+exports.run = async (client,message,args) => {
+    const embed = new MessageEmbed();
+    try{
+        console.log("Test1");
+        let mes = client.songRecs.get(message.author.id);
+        embed.setTitle(`${message.author.id}'s Song Recommendation`);
+        embed.setDescription(mes);
+        message.channel.send({embeds:[embed]});
```

Comment:
You can use try{}catch{}finally {message.channel.send({embeds:[embed]})} to avoid repeating some code